### PR TITLE
ARM64 Ubuntu 20.04 Dockerfile for GitHub Actions

### PR DESCRIPTION
**Pull request by fengyuentau**
Images already built based on the Dockerfile in this PR: https://hub.docker.com/r/yuentau/ocv_ubuntu

### CODE REVIEW

**Comment by asenyaev** [comment]
:+1:

**Comment by fengyuentau** [comment]
Some packages are added from the Dockerfile in @asenyaev 's PR https://github.com/opencv-infrastructure/opencv-worker-config/pull/1. The new image is pushed to quay.io as well.

File: images/ubuntu-github-actions-arm64--20.04/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:20.04
+
+RUN \
+   apt-get update && \
+   DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+     cmake build-essential ccache ninja-build git wget curl zip unzip mc nano time file\
```
Comment (asenyaev) [inline images/ubuntu-github-actions-arm64--20.04/Dockerfile]: Could you add `rsync` here for the extended DNN testing?

I've already rebuilt image with it and pushed to the docker registry for PR testing.

Comment (fengyuentau) [reply images/ubuntu-github-actions-arm64--20.04/Dockerfile]: Sure. Thanks for rebuilding the image and pushing to docker registry.

File: images/ubuntu-github-actions-arm64--20.04/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+   apt-get update && \
+   DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+     libva-dev \
+     libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev \
+         gstreamer1.0-plugins-good gstreamer1.0-plugins-ugly gstreamer1.0-libav \
+     libopenni2-dev \
+     libdc1394-22-dev \
+     libgphoto2-dev \
+     && \
+   mkdir /opencv /opencv-build /opencv-contrib-build && \
```
Comment (asenyaev) [inline images/ubuntu-github-actions-arm64--20.04/Dockerfile:35]: Need to add `/opencv_contrib` folder for OpenCV Contrib builds.

**Comment by asenyaev** [comment]
@fengyuentau, I have pushed dockerfiles from our PRs to a dedicated repo [opencv-infrastructure/opencv-gha-dockerfile](https://github.com/opencv-infrastructure/opencv-gha-dockerfile) including the change which I mentioned above, and pushed it to the docker registry.

If you have any updates, please feel free to update dockerfiles in the new repository. I'm closing this PR.

**Pull request closed without merging**